test_output/help/bmcs.txt

```diff
@@ -1,21 +1,24 @@
 knife bmcs  --help
 
-Available bmcs subcommands: (for details, knife SUB-COMMAND --help)
-
-** BMCS COMMANDS **
-knife bmcs ad list (options)
-knife bmcs compartment list (options)
-knife bmcs image list (options)
-knife bmcs server create (options)
-knife bmcs server delete (options)
-knife bmcs server list (options)
-knife bmcs server show (options)
-knife bmcs shape list (options)
-knife bmcs subnet list (options)
-knife bmcs vcn list (options)
-
-
-
-+++ STDERR +++
-
-FATAL: Cannot find subcommand for: 'bmcs --help'
+knife bmcs
```

Comment:
Shouldn't the new output be the same as the old output, except replacing 'bmcs' with 'oci' ?

Reply:
Good point - this command was added simply to show a deprecation warning when running "knife bmcs" in order to encourage existing users to switch to oci. That said, it's a bit heavy handed - when not using --help with the command it will no longer list the subcommands, and when using --help, as you point out, it shows a list of options that are not meaningful. I've looked into ways to improve this, but it doesn't appear that there's a good *supported* way to do that (that is, that would not depend on Chef implementation details that might break in different versions). So I will remove this change.